app/jobs/reports/verification_errors_report.rb

```diff
@@ -0,0 +1,62 @@
+require 'identity/hostdata'
+require 'csv'
+
+module Reports
+  class VerificationErrorsReport < BaseReport
+    REPORT_NAME = 'verification-errors-report'.freeze
+
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      total_limit: 1,
+      key: -> { "#{REPORT_NAME}-#{arguments.first}" },
+    )
+
+    def perform(_date)
+      csv_reports = []
+      configs = IdentityConfig.store.verification_errors_report_configs
+      configs.each do |report_hash|
+        issuers = report_hash['issuers']
+        report = verification_errors_data_for_issuers(issuers)
+        csv_reports << report
+      end
+      csv_reports
+    end
+
+    private
+
+    def verification_errors_data_for_issuers(issuers)
+      csv = CSV.new('', row_sep: "\r\n")
+      csv << %w[uuid welcome_view_at error_code]
+      issuers.each do |issuer|
+        transaction_with_timeout do
+          rows = ::VerificationErrorsReport.call(issuer, 24.hours.ago)
+          rows.each do |row|
+            csv << [row['uuid'], row['welcome_view_at'].iso8601, ial2_error_code(row)]
+          end
+        end
+      end
+      data = csv.string
+      save_report("%{REPORT_NAME}.{issuers.join('.')}", data, extension: 'csv')
```

Comment:
1. this going to have literal `%{REPORT_NAME}` in the filename, I think we want `#{REPORT_NAME}`
2. I just realized that this structure groups together issues... that's going to be a very long name, would it make sense to make one report per issuer?

Reply:
I guess.  the deleted accounts report had an agency with two issuers so i just followed that format but I don't see the need for it here.